assets/learning-mode/learning-mode.editor.js

```diff
@@ -4,21 +4,21 @@
  */
 import { getQueryArgs, addQueryArgs } from '@wordpress/url';
 
-window.addEventListener( 'locationchange', redirectToCourseThemeOverride );
-window.addEventListener( 'popstate', redirectToCourseThemeOverride );
+window.addEventListener( 'locationchange', redirectToLearningModeOverride );
+window.addEventListener( 'popstate', redirectToLearningModeOverride );
 
 /**
  * Reload the page when opening or closing a course theme template, to ensure the active theme styles are not loaded.
  */
-function redirectToCourseThemeOverride() {
+function redirectToLearningModeOverride() {
```

Comment:
Note when resolving merge conflicts that all this was removed in https://github.com/Automattic/sensei/pull/5563/files#diff-a979e723b84d8455f85f71e9b8d44e0969baa86b3e082a997b5742681d119579

Reply:
Okay got it 👌